packages/astro/e2e/view-transitions.test.js

```diff
@@ -168,8 +168,6 @@ test.describe('View Transitions', () => {
 	});
 
 	test('Moving from a page without ViewTransitions w/ back button', async ({ page, astro }) => {
-		const loads = collectLoads(page);
```

Comment:
@martrapp do you know if this was required for the test (maybe doing a side-effect?)

Reply:
Hi @florian-lefebvre, thank you for asking! The loads value was indeed unused. Now it has a purpose ;-)